internal/pkg/manifest/env.go

```diff
@@ -328,13 +330,46 @@ func (cfg *environmentHTTPConfig) loadLBConfig(env *config.CustomizeEnv) {
 	cfg.Public.Certificates = env.ImportCertARNs
 }
 
-type publicHTTPConfig struct {
-	Certificates []string `yaml:"certificates,omitempty"`
+// ALBSecurityGroupsConfig represents security group configuration settings for an ALB.
+type ALBSecurityGroupsConfig struct {
```

Comment:
nit: can we move this type after `PublicHTTPConfig`

Reply:
Did both ways testing, going from `PublicLoadBalancerSecurityGroup` to `PublicHTTPLoadBalancerSecurityGroup` and back the other way successfully creates the resource, updates the load balancer, and then deletes the unneeded resource in order.